README.md

```diff
@@ -79,6 +79,12 @@ it is, please follow the non-technical [User Guide](https://cinnamon.github.io/k
 
 ### For developers
 
+#### System requirements
+
+1. Python 3.10>=
+2. [homebrew](https://brew.sh/)
+3. [Docker](https://www.docker.com/)
```

Comment:
These should be optional as we provide options to install from source without docker and homebrew.

Reply:
if `unstructured` is going to remain a dependency of `kotaemon`, then it's system dependencies should be documented as well - https://docs.unstructured.io/open-source/introduction/quick-start